### ISSUE TITLE
Ribbon Cable Trigger on the attenuator/ESP32 site

### ISSUE THREAD

**Issue by Aces6469** [gpstar81/GPStar-proton-pack#388]
### What would you like to add in terms of software changes?

Surprised it hasn't been considered yet or I'm just blind. Could we get a trigger to have the ribbon cable open or closed on the site? I'm one of the people that have a static cable clamp so it'd be nice to have access to the feature.

### Would this request involve any specific hardware?

ESP32/Attenuator

### Homework Completed

- [X] I did my part!

**Comment by DustinGrau** [comment]
Just so I'm clear, are you asking for a button on the web UI to manually trigger the ribbon cable alarm effects? If so, there's plans to open up the full APIs so we can trigger any event on the pack or wand also via the ESP32 device, but that's a ways off. That feature needs to be implemented before we can open up other actions. The other one requested is triggering stream mode changes via the web UI, but again requires this same underlying feature.

**Comment by Aces6469** [comment]
Yep, hit the nail on the head.

**Comment by nomakewan** [comment]
This is in as of commit [6ccb616](https://github.com/gpstar81/GPStar-proton-pack/pull/468/commits/6ccb616d7611b4c9e5e8741f4277ae5ab3ef86a9) and is slated for inclusion in 6.1.1.

The way it works is that if you have "Respond to Ribbon Cable Alarm" disabled on the Proton Pack (that is, you disabled the physical switch), you can tap the ribbon cable on the web UI's graphical interface in order to toggle between the alarm on and the alarm off. Alternatively, a web API is provided to do the same at `/pack/cable/on` and `/pack/cable/off`.
